patches/chromium/allow_in_process_windows_to_have_different_web_prefs.patch

```diff
@@ -0,0 +1,132 @@
++    delegate_->WebContentsCreatedWithFullParams(this, render_process_id, opener->GetRoutingID(), params, new_contents_impl);
++  }
++
+   new_contents_impl->GetController().SetSessionStorageNamespace(
+       partition_id, session_storage_namespace);
+ 
+@@ -3094,12 +3098,6 @@ RenderFrameHostDelegate* WebContentsImpl::CreateNewWindow(
+     AddDestructionObserver(new_contents_impl);
+   }
+ 
+-  if (delegate_) {
+-    delegate_->WebContentsCreatedWithFullParams(this, render_process_id,
+-                                                opener->GetRoutingID(),
+-                                                params, new_contents_impl);
+-  }
+-
+   for (auto& observer : observers_) {
+     observer.DidOpenRequestedURL(new_contents_impl, opener, params.target_url,
+                                  params.referrer.To<Referrer>(),
+diff --git a/content/public/common/common_param_traits_macros.h b/content/public/common/common_param_traits_macros.h
+index 5ec7bb1d1abc12b957690c04046cafbf4989000c..2019e1daf4d30eb40439d738934f6a567acd0c50 100644
+--- a/content/public/common/common_param_traits_macros.h
++++ b/content/public/common/common_param_traits_macros.h
+@@ -132,6 +132,27 @@ IPC_STRUCT_TRAITS_BEGIN(content::WebPreferences)
+   IPC_STRUCT_TRAITS_MEMBER(webgl2_enabled)
+   IPC_STRUCT_TRAITS_MEMBER(pepper_3d_enabled)
+   IPC_STRUCT_TRAITS_MEMBER(record_whole_document)
++
++  IPC_STRUCT_TRAITS_MEMBER(preloads)
++  IPC_STRUCT_TRAITS_MEMBER(disable_electron_site_instance_overrides)
++  IPC_STRUCT_TRAITS_MEMBER(background_color)
++  IPC_STRUCT_TRAITS_MEMBER(opener_id)
++  IPC_STRUCT_TRAITS_MEMBER(context_isolation)
++  IPC_STRUCT_TRAITS_MEMBER(enable_remote_module)
++  IPC_STRUCT_TRAITS_MEMBER(guest_instance_id)
++  IPC_STRUCT_TRAITS_MEMBER(hidden_page)
++  IPC_STRUCT_TRAITS_MEMBER(offscreen)
++  IPC_STRUCT_TRAITS_MEMBER(preload)
++  IPC_STRUCT_TRAITS_MEMBER(native_window_open)
++  IPC_STRUCT_TRAITS_MEMBER(node_integration)
++  IPC_STRUCT_TRAITS_MEMBER(node_integration_in_worker)
++  IPC_STRUCT_TRAITS_MEMBER(node_leakage_in_renderers)
++  IPC_STRUCT_TRAITS_MEMBER(node_integration_in_sub_frames)
++  IPC_STRUCT_TRAITS_MEMBER(enable_spellcheck)
++  IPC_STRUCT_TRAITS_MEMBER(enable_plugins)
++  IPC_STRUCT_TRAITS_MEMBER(enable_websql)
++  IPC_STRUCT_TRAITS_MEMBER(webview_tag)
++
+   IPC_STRUCT_TRAITS_MEMBER(flash_3d_enabled)
+   IPC_STRUCT_TRAITS_MEMBER(flash_stage3d_enabled)
+   IPC_STRUCT_TRAITS_MEMBER(flash_stage3d_baseline_enabled)
+diff --git a/content/public/common/web_preferences.cc b/content/public/common/web_preferences.cc
+index b543f98a34266efdb771482c5717042700a47ed0..4ab4ae9855ad470fe8c79de7a294b6630a5f60f6 100644
+--- a/content/public/common/web_preferences.cc
++++ b/content/public/common/web_preferences.cc
+@@ -166,6 +166,27 @@ WebPreferences::WebPreferences()
+       navigate_on_drag_drop(true),
+       v8_cache_options(blink::mojom::V8CacheOptions::kDefault),
+       record_whole_document(false),
++
++      preloads(base::EmptyString()),
```

Comment:
`preloads` vs `preload`?

Reply:
I'm preserving the distinction already present between `kPreloadScript` and `kPreloadScripts`